### PR TITLE
Orientation sphinx documentation

### PR DESCRIPTION
**Pull request by kperrynrel**
## Description

This PR handles documentation for the sunny day mask for both fixed tilt and single axis tracking systems functions.

*Thank you for your contribution! Please provide a brief description of the problem and the proposed solution or new feature (if not already fully described in a linked issue)*

## Checklist

The following items must be addressed before the code can be merged. 
Please don't hesitate to ask for help if you are unsure of how to accomplish any of the items. 
*You are free to remove any checklist items that do not apply or add additional items that are 
not on this list*

- [x] Clearly documented all new API functions with [PEP257](https://www.python.org/dev/peps/pep-0257/) and [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings
- [x] Adds description and name entries in the appropriate "what's new" file 
      in [`docs/whatsnew`](https://github.com/pvlib/pvanalytics/tree/master/docs/whatsnew) 
      for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` 
      or this Pull Request with `` :pull:`num` ``. Includes contributor name 
      and/or GitHub username (link with `` :ghuser:`user` ``).
- [ X ] Non-API functions clearly documented with docstrings or comments as necessary
- [x] Pull request is nearly complete and ready for detailed review

### CODE REVIEW

File: docs/examples/fixed-nrel.py
Comment (kanderso-nrel) [inline docs/examples/fixed-nrel.py:15]: Mention the relevant OEDI system_id

File: docs/examples/tracking-nrel.py
Comment (kanderso-nrel) [inline docs/examples/tracking-nrel.py:15]: Mention the relevant OEDI system_id

**Review by kanderso-nrel — COMMENTED** [review]
@kperrynrel both this PR and #146 add a SERF data file.  Should we merge #146 first and then update this PR to use that dataset?

**Comment by kperrynrel** [comment]
@kanderso-nrel Yeah I'm for us waiting until #146 is merged before merging this guy.

File: docs/examples/tracking-nrel.py
Comment (kperrynrel) [reply docs/examples/tracking-nrel.py:15]: Updated in commit https://github.com/pvlib/pvanalytics/pull/148/commits/0c8c07e829f219dcc2de10a0b1d9b5eb8f8af930

File: docs/examples/fixed-nrel.py
Comment (kperrynrel) [reply docs/examples/fixed-nrel.py]: Updated in commit https://github.com/pvlib/pvanalytics/pull/148/commits/0c8c07e829f219dcc2de10a0b1d9b5eb8f8af930

File: docs/examples/tracking-nrel.py
Comment (kanderso-nrel) [inline docs/examples/tracking-nrel.py:11]: Wording nitpick -- I think it's probably a good idea to avoid using "clear-sky" and "sunny day" as synonyms.  Clear-sky (at least in the pvlib world) has some association with smoothness, but I think these functions would not take issue with a jagged input curve so long as the jaggedness is riding on top of the quartic/quadratic profile they're looking for. 

The tracking function has the additional requirement that the input *isn't* a good match to a fixed-tilt profile, so to my mind it's more of a check that the system is tracking properly, and the identification of sunny days was more of an implementation detail than a specific goal.  Not sure how relevant that is here though.

**Review by kanderso-nrel — COMMENTED** [review]
What should we do with the SERF data file?  It's way more data than this PR needs, so maybe pare it down to just those few days with the expectation that #146 might overwrite it later with a longer dataset?

File: docs/examples/tracking-nrel.py
Comment (kperrynrel) [reply docs/examples/tracking-nrel.py:11]: I changed the wording to 'sunny day' only in https://github.com/pvlib/pvanalytics/pull/148/commits/76c4de1b0b66a64a618800c58fe174ace294d98e

**Review by kanderso-nrel — APPROVED** [review]
OK with me!

**Review by cwhanse — APPROVED** [review]

**Pull request merged**
